packages/core/base/src/models/types.ts

```diff
@@ -16,6 +16,9 @@ export enum customHeaders {
 
 type theme = "light" | "dark";
 
+export type paymentMethods = "ETH";
```

Comment:
I think we should add "fiat" (category for Credit card, debit card, and whatever we add in the future) as an option, and make it the default if omitted